lib/runners/processor/code_sniffer.rb

```diff
@@ -29,10 +29,12 @@ class Processor::CodeSniffer < Processor
 
     register_config_schema(name: :code_sniffer, schema: Schema.runner_config)
 
+    DEFAULT_CONFIG_FILE = (Pathname(Dir.home) / "sider_recommended_code_sniffer.xml").to_path.freeze
+
     DefaultOptions = _ = Struct.new(:standard, :extensions, :target, keyword_init: true)
     OPTIONS_CAKE_PHP = DefaultOptions.new(standard: "CakePHP", extensions: "php", target: "app/").freeze
     OPTIONS_SYMFONY = DefaultOptions.new(standard: "Symfony", extensions: "php", target: "src/").freeze
-    OPTIONS_PSR2 = DefaultOptions.new(standard: "PSR2", extensions: "php", target: "./").freeze
+    OPTIONS_PSR2 = DefaultOptions.new(standard: DEFAULT_CONFIG_FILE, extensions: "php", target: "./").freeze
```

Comment:
Is this OK to remain the constant name as `OPTIONS_PSR2` even though the `standard` is not `"PSR2"`?

Reply:
@sleeeto It's confusing. Please rename the constant.

```suggestion
    OPTIONS_DEFAULT = DefaultOptions.new(standard: DEFAULT_CONFIG_FILE, extensions: "php", target: "./").freeze
```

Reply:
OK, I renamed it.